app/sim_test.go

```diff
@@ -261,8 +247,13 @@ func TestAppImportExport(t *testing.T) {
 		{app.keys[poststypes.StoreKey], newApp.keys[poststypes.StoreKey], [][]byte{}},
 		{app.keys[reportstypes.StoreKey], newApp.keys[reportstypes.StoreKey], [][]byte{}},
 		{app.keys[reactionstypes.StoreKey], newApp.keys[reactionstypes.StoreKey], [][]byte{}},
+
+		{app.keys[wasm.StoreKey], newApp.keys[wasm.StoreKey], [][]byte{}},
 	}
 
+	// delete persistent tx counter value
+	ctxA.KVStore(app.keys[wasm.StoreKey]).Delete(wasmtypes.TXCounterPrefix)
```

Comment:
Wasm simulation does not check TxCounter, it must be deleted manually here.
See:
https://github.com/CosmWasm/wasmd/blob/v0.30.0/x/wasm/keeper/ante.go#L24